assets/sass/components/_home-content.scss

```diff
@@ -110,11 +106,6 @@
   }
 }
 
-.home-content__four-items > a:hover .content-link--series,
-.home-content__four-items > a:focus .content-link--series {
-  color: govuk-colour('white');
-}
-
```

Comment:
No longer required as the new icon applies the same SASS properties as all the other icons